README.md

```diff
@@ -32,6 +33,31 @@ Read the [Changelog](https://github.com/Microsoft/vscode-go/blob/master/CHANGELO
 - [Code of Conduct](#code-of-conduct)
 - [License](#license)
 
+## We are moving!
+
+🏡 🚚 🏡 
+
+Our new home will be https://github.com/golang/vscode-go.
+
+As the extension grows in popularity and supports more features, it requires more maintenance and support.
+Since the introduction of Go modules, we have been collaborating with the Go team for module support.
+In 2019, we added support for `gopls`, the [Go language server](https://golang.org/s/gopls/README.md) which has
+resulted in closer collaboration with the Go team. More exciting work is on the horizon with the
+[Go debug adapter service](https://github.com/go-delve/delve/tree/master/service/dap) in delve which will help
```

Comment:
Capitalize Delve?

Reply:
And just say "debug adapter service" instead of "Go debug adapter service"

Reply:
I wouldnt captialize delve, it is a tool like gopls

Reply:
Instead of "debug adapter service", am thinking more like "implementation of debug adapter protocol in delve". Thoughts?

Reply:
Re: capitalization, I was just thinking that based on the docs here: https://github.com/go-delve/delve#about-delve.

I think either way works - I imagine most people aren't familiar with DAP so as long as we say debugging I think it's fine.